frontend/src/choreoComponents/ChoreoDetail.tsx

```diff
@@ -4,62 +4,278 @@ import {
+                                                        title="up"
+                                                        sx={{color:"lightgrey"}} onClick={() => onMoveUp(move.id, index)}>
+                                                        <ArrowUpwardIcon />
+                                                    </IconButton>
+                                                    <IconButton
+                                                        title="down"
+                                                        sx={{color:"lightgrey"}} onClick={() => onMoveDown(move.id, index)}>
+                                                        <ArrowDownwardIcon />
+                                                    </IconButton>
+                                                </Box>
+
+                                    </CardContent>
+                                </Card>
+                            )}
+                            <Grid item xs={12}>
+                                <Grid container spacing={2} justifyContent="space-evenly" mt={2} mb={2}>
+                                    <Button
+                                        id="galleryBtn"
+                                        type="button"
+                                        variant="contained"
+                                        onClick={() => {
+                                            navigate("/choreo")
+                                        }}
+                                        startIcon={<ArrowBackIcon/>}>
+                                        Back to gallery
+                                    </Button>
+                                    <Button
+                                        id="galleryBtn"
+                                        variant="contained"
+                                        color="success"
+                                        startIcon={<AddIcon/>}
+                                        onClick={handleOpen}>
+                                        Add Move
+                                    </Button>
+                                    <Button
+                                        id="galleryBtn"
+                                        type="submit"
+                                        variant="contained"
+                                        startIcon={<SaveIcon/>}
+                                    >
+                                        Save Update
+                                    </Button>
+                                    <Button
+                                        id="deleteBtn"
+                                        type="submit"
+                                        variant="outlined"
+                                        color="error"
+                                        startIcon={<DeleteIcon/>}
+                                        onClick={deleteChoreoOnClick}
+                                    >
```

Comment:
auch diese Buttons hier könnte man auslagern, evtl. kannst du ihnen sogar die Funktionen mitgeben und direkt in der ausgelagerten Komponente per Click ausführen. Sie bräuchten dafür nur den state